Framework/PythonInterface/mantid/geometry/src/Exports/DetectorInfo.cpp

```diff
@@ -54,9 +54,10 @@ void export_DetectorInfo() {
            "Sets all mask flags to false (unmasked).")
 
       .def("isEquivalent", &DetectorInfo::isEquivalent,
-           (arg("self"), arg("other")), "Returns True if the content of this "
-                                        "detector is equivalent to the content "
-                                        "of the other detector.")
+           (arg("self"), arg("other")),
```

Comment:
You shouldn't have to apply clang formatting changes to code that has not been changed by this PR. It's probably worth checking if this is resolved on master then merging this branch instead (this will make for a much cleaner git history)